src/Microsoft.DotNet.Interactive.Document/NotebookFileFormatHandler.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.DotNet.Interactive.dib;
+using Microsoft.DotNet.Interactive.Ipynb;
+
+namespace Microsoft.DotNet.Interactive
+{
+    public static class NotebookFileFormatHandler
```

Comment:
I'd say the word "handler" is misleading here.

For context and discoverability, I'd maybe name this class `InteractiveDocument`.